source/includes/_endpoint-dataset.md

```diff
@@ -565,3 +565,23 @@ PATCH the "expression" attribute to modify. An empty "expression" object, like
 ##### Decks
 
 ##### Permissions
+
+##### Primary key
+
+`/datasets/{id}/pk`
+
+Setting a primary key on a dataset causes updates (particularly streamed
+updates) mentioning existing rows to be updated instead of new rows being
+inserted.
+
+GET on this resource returns a Shoji Entity with a "pk"" attribute in its body,
+which is a list. The "pk" attribute indicates the variable ids of the columns in
+the dataset which comprise the primary key.
+
+POST the "pk" attribute to modify the pk. The "pk" attribute should be a list
+containing zero or more variable ids or variable aliases.  Setting the pk to
```

Comment:
Does it accept URLs?

How do we feel about accepting aliases? Seems like it would be a nice convenience here, but really more of a convenience everywhere else in the API. I can't remember if there was a reason we don't accept them anywhere else.